packages/ui/src/App.tsx

```diff
@@ -18,42 +17,39 @@ const App: React.FC = () => {
                 <BrowserRouter>
                     <Routes>
                         <Route path="/" element={<HomePage />} />
-                        <Route
-                            path="/dashboard"
-                            element={
-                                <PrivateRoute>
-                                    <Dashboard />
-                                </PrivateRoute>
-                            }
-                        />
-                        <Route
-                            path="/flags"
-                            element={
-                                <PrivateRoute>
-                                    <Flags />
-                                </PrivateRoute>
-                            }
-                        />
-                        <Route
-                            path="/segments"
-                            element={
-                                <PrivateRoute>
-                                    <Segments />
-                                </PrivateRoute>
-                            }
-                        />
-                        <Route
-                            path="/settings/apikeys"
-                            element={
-                                <PrivateRoute>
-                                    <ApiKeys />
-                                </PrivateRoute>
-                            }
-                        />
-                        <Route element={<AuthLayout />}>
+
+                        <Route element={<DashboardLayout />}>
```

Comment:
In this way the DashboardLayout will be unprotected. You could move the <DashboardLayout inside the privateRoute in each route.